tools/schemacode/bidsschematools/tests/test_validator.py

```diff
@@ -86,12 +86,12 @@ def test_write_report(tmp_path):
     ]
     validation_result["schema_listing"] = [
         {
-            "regex": ".*?/sub-(?P<subject>[0-9a-zA-Z]+)/"
-            "(|ses-(?P<session>[0-9a-zA-Z]+)/)anat/sub-(?P=subject)"
-            "(|_ses-(?P=session))(|_acq-(?P<acquisition>[0-9a-zA-Z]+))"
-            "(|_ce-(?P<ceagent>[0-9a-zA-Z]+))"
-            "(|_rec-(?P<reconstruction>[0-9a-zA-Z]+))"
-            "(|_run-(?P<run>[0-9a-zA-Z]+))"
+            "regex": ".*?/sub-(?P<subject>[0-9a-zA-Z+]+)/"
+            "(|ses-(?P<session>[0-9a-zA-Z+]+)/)anat/sub-(?P=subject)"
+            "(|_ses-(?P=session))(|_acq-(?P<acquisition>[0-9a-zA-Z+]+))"
+            "(|_ce-(?P<ceagent>[0-9a-zA-Z+]+))"
+            "(|_rec-(?P<reconstruction>[0-9a-zA-Z+]+))"
+            "(|_run-(?P<run>[0-9a-zA-Z+]+))"
```

Comment:
This one doesn't fit. `run` takes an index, not a label. I'm not sure why it currently accepts an alphanumeric label.

Reply:
This is not generated from anything, but was hand-written to test writing out a report.

Reply:
Good eye! I guess it was a bug, let's fix

```suggestion
            "(|_run-(?P<run>[0-9]+))"
```

?